README.md

```diff
@@ -12,69 +12,69 @@ Search [Crystal Shards](https://crystalshards.xyz) or follow announcements [Crys
-  * [Unofficial](#unofficial)
-* [Resources](#resources)
-  * [Official Documentation Translations](#official-documentation-translations)
-* [Services and Apps](#services-and-apps)
-* [Tools](#tools)
-  * [DevOps](#devops)
-  * [Editor Plugins](#editor-plugins)
-  * [LSP Language Server Protocol Implementations](#lsp-language-server-protocol-implementations)
-  * [Shell Plugins](#shell-plugins)
+- [Awesome Crystal](#awesome-crystal)
+  - [Algorithms and Data structures](#algorithms-and-data-structures)
+  - [Blockchain](#blockchain)
+  - [C bindings](#c-bindings)
+  - [Caching](#caching)
+  - [CLI Builders](#cli-builders)
+  - [CLI Utils](#cli-utils)
+  - [Code Analysis and Metrics](#code-analysis-and-metrics)
+  - [Compression](#compression)
+  - [Configuration](#configuration)
+  - [Converters](#converters)
+  - [Cryptography](#cryptography)
+  - [Data Formats](#data-formats)
+  - [Data Generators](#data-generators)
+  - [Database Drivers/Clients](#database-driversclients)
+  - [Database Tools](#database-tools)
+  - [Debugging](#debugging)
+  - [Dependency Injection](#dependency-injection)
+  - [Email](#email)
+  - [Environment Management](#environment-management)
+  - [Examples and funny stuff](#examples-and-funny-stuff)
+  - [Framework Components](#framework-components)
+  - [Game Development](#game-development)
+  - [GUI Development](#gui-development)
+  - [HTML Builders](#html-builders)
+  - [HTML/XML Parsing](#htmlxml-parsing)
+  - [HTTP](#http)
+  - [Image processing](#image-processing)
+  - [Implementations/Compilers](#implementationscompilers)
+  - [Internationalization](#internationalization)
+  - [Logging and monitoring](#logging-and-monitoring)
+  - [Machine Learning](#machine-learning)
+  - [Markdown/Text Processors](#markdowntext-processors)
+  - [Misc](#misc)
+  - [Network Protocols](#network-protocols)
+  - [Networking](#networking)
+  - [ORM/ODM Extensions](#ormodm-extensions)
+  - [Package Management](#package-management)
+  - [Processes and Threads](#processes-and-threads)
+  - [Project Generators](#project-generators)
+  - [Queues and Messaging](#queues-and-messaging)
+  - [Routing](#routing)
+  - [Scheduling](#scheduling)
+  - [Science and Data analysis](#science-and-data-analysis)
+  - [Search](#search)
+  - [Serverless Computing](#serverless-computing)
+  - [System](#system)
+  - [Task management](#task-management)
+  - [Template Engine](#template-engine)
+  - [Testing](#testing)
+  - [Third-party APIs](#third-party-apis)
+  - [Validation](#validation)
+  - [Web Frameworks](#web-frameworks)
+- [Community](#community)
+  - [Unofficial](#unofficial)
+- [Resources](#resources)
+  - [Official Documentation Translations](#official-documentation-translations)
+- [Services and Apps](#services-and-apps)
+- [Tools](#tools)
+  - [DevOps](#devops)
+  - [Editor Plugins](#editor-plugins)
+  - [LSP Language Server Protocol Implementations](#lsp-language-server-protocol-implementations)
+  - [Shell plugins](#shell-plugins)
```

Comment:
Probably should revert this change.

Reply:
Whoa.  What the heck? I made a fresh fork, and added one line and one line only.  *sigh*

I'm going to kill this PR completely, and start over in a new branch, and see if I can figure out what the heck happened there.